src/styles/index.scss

```diff
@@ -1,3 +1,85 @@
+$day-size: 100px;
```

Comment:
The file is named `index.scss`, but according to the task requirements, it should be named `main.scss`. Please rename the file to ensure consistency with the project structure.